### PR TITLE
Address golint warning for package plugin and schedule

### PR DESCRIPTION
**Pull request by candysmurf**

### CODE REVIEW

**Comment by danielscottt** [comment]
This looks pretty good, but I'm not sure how helpful the `// const defines constants` and the `// scheduler vars`-style comments are.  I think the vet/lint errors you see here are with respect to commenting on the exported items, be they functions or variables.

File: scheduler/scheduler.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ import (
 	"github.com/intelsdi-x/snap/scheduler/wmap"
 )
 
+// Schedule var variables
```
Comment (jcooklin) [inline scheduler/scheduler.go]: You could describe how HandlerRegistrationName is used to register a gomit event handler.

File: scheduler/task.go [hunk 2]
```diff
@@ -35,11 +35,13 @@ import (
 	"github.com/intelsdi-x/snap/scheduler/wmap"
 )
 
+// const defines constants
```
Comment (jcooklin) [inline scheduler/task.go]: A description of what the DefaultDeadlineDuration and DefaultStopOnFailure do should be provided.

**Comment by danielscottt** [comment]
LGTM

**Comment by danielscottt** [comment]
@candysmurf Are you ready to merge this one?

**Pull request merged**